saleor/graphql/checkout/types.py

```diff
@@ -172,6 +193,9 @@ class Checkout(CountableDjangoObjectType):
     is_shipping_required = graphene.Boolean(
         description="Returns True, if checkout requires shipping.", required=True
     )
+    is_click_and_collect = graphene.Boolean(
+        description="Returns True, if checkout is click and collect"
```

Comment:
"Returns true, if click and collect is chosen as a delivery method for this checkout."

Reply:
Done.